vector/src/main/res/values/styles.xml

```diff
@@ -429,7 +429,7 @@
         <item name="android:actionBarDivider">@null</item>
 
         <!-- switch color -->
-        <item name="colorControlActivated">@color/vector_green_color</item>
+        <item name="colorControlActivated">@android:color/white</item>
```

Comment:
Isn't this change too strong?

Reply:
I was going to remove the style changes here, there was a reason it was in there but I forget it now.